sdk/translation/ai-translation-document-rest/package.json

```diff
@@ -0,0 +1,124 @@
+{
+  "name": "@azure-rest/ai-translation-document",
+  "sdk-type": "client",
+  "author": "Microsoft Corporation",
+  "description": "A generated SDK for DocumentTranslationClient.",
+  "version": "1.0.0-beta.1",
+  "keywords": [
+    "node",
+    "azure",
+    "cloud",
+    "typescript",
+    "browser",
+    "isomorphic",
+    "translate",
+    "translation"
+  ],
+  "license": "MIT",
+  "main": "dist/index.js",
+  "module": "./dist-esm/src/index.js",
+  "types": "./types/ai-translation-document-rest.d.ts",
+  "homepage": "https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/translation/ai-translation-document-rest/README.md",
+  "repository": "github:Azure/azure-sdk-for-js",
+  "bugs": {
+    "url": "https://github.com/Azure/azure-sdk-for-js/issues"
+  },
+  "files": [
+    "dist/",
+    "dist-esm/src/",
+    "types/ai-translation-document.d.ts",
+    "README.md",
+    "CHANGELOG.md",
+    "LICENSE"
+  ],
+  "engines": {
+    "node": ">=18.0.0"
+  },
+  "//sampleConfiguration": {
+    "productName": "Azure Document Translation Service",
+    "productSlugs": [
+      "azure",
+      "azure-cognitive-services",
+      "azure-translator"
+    ],
+    "requiredResources": {
+      "Azure Cognitive Services instance": "https://docs.microsoft.com/azure/cognitive-services/cognitive-services-apis-create-account"
+    }
+  },
+  "dependencies": {
+    "@azure/core-rest-pipeline": "^1.8.0",
+    "@azure/logger": "^1.0.0",
+    "@typespec/ts-http-runtime": "1.0.0-alpha.20240314.2",
```

Comment:
@bterlson 👋 just curious - when is a beta planned for this? I'd feel _marginally_ better although realistically I get that it does not matter in this case

Reply:
This package should not be a dependency, please remove it. Anything being used from this package is also accessible in an Azure Core package -- please use those instead.

Reply:
I was wondering how this was added too if this package is mostly generated.

Reply:
yeah, I am not sure. If it was just the unbranded package, that's conceivable if misconfigured, but seems strange to have both unbranded and Azure packages in the dependencies

Reply:
Deleted